src/i18n/util.ts

```diff
@@ -104,3 +104,13 @@ export function useTranslations(
 		return str;
```

Comment:
Could we refactor `useTranslations` to use the new function and avoid duplication?

```js
export function useTranslations(
	Astro: Readonly<AstroGlobal>
): (key: UIDictionaryKeys) => string | undefined {
	const lang = getLanguageFromURL(Astro.url.pathname) || 'en';
	return useTranslationsForLang(lang);
}
```

Reply:
Agree, smart suggestions!